### PR TITLE
XWIKI-5: Make Chart Configurable with external solr

### PR DESCRIPTION
**Pull request by ASHISH932**

### CODE REVIEW

**Comment by slayerjain** [comment]
Looks fine.

File: values.yaml [hunk 1]
```diff
@@ -61,6 +61,14 @@ externalDB:
   database: ''
   host: ''
 
+##
+## To use external solr enable solr, provide host and port
+##
+solr:
+  enabled: false
+  host: ""
```
Comment (nehagup) [inline values.yaml]: Let's not keep it an empty string. 
Add default host and port.

File: values.yaml [hunk 2]
```diff
@@ -61,6 +61,14 @@ externalDB:
...
+  port: ""
```
Comment (nehagup) [inline values.yaml]: Same as above.

File: values.yaml (hunk 1 above)
Comment (slayerjain) [reply values.yaml]: Yeah, this way it'll look cleaner, and give users more clarity about the data types.

**Pull request merged**